### PR TITLE
Add an acknowledgement section to the about.html page #3830

### LINKED ISSUE #3830: Add an acknowledgement section to the about.html page
We can thank the following parties under this section:
CDTL (TEG grants)
LIFT grant
YourKit
GSoC

### PR DESCRIPTION
**Pull request by junhaoyap**
Fixes #3830

### CODE REVIEW

**Comment by junhaoyap** [comment]
screen shot 2015-06-28 at 11 42 03 pm

Preview of how it looks like, trying to make it consistent with the "about us" below, note that the copy is taken from our readme.md but the copy below in "about us" has been slightly changed to reflect the acknowledgements and not be repetitive.

When we eventually port these pages into bootstrap and have them be responsive also, we might want to change the layout slightly.

**Comment by junhaoyap** [comment]
@damithc could you take a look at this? 

Perhaps comment on the copy in case I made any undesirable change on the copy of the "about us" section.

Or maybe we want the "acknowledgements" section to be below the "about us" section.

**Comment by junhaoyap** [comment]
Also note that there is inline css atm, I plan to remove them (abstract in classes and use CSS files) when I move these pages to bootstrap and make them mobile friendly.

File: src/main/webapp/about.html [hunk 1]
```diff
@@ -73,20 +73,44 @@
...
-          <span class="bold">TEAMMATES</span> is a joint effort by the team listed below and brought
-          to you with the kind support of the <a href="http://www.comp.nus.edu.sg/">School of
-            Computing</a> and the <a href="http://www.cdtl.nus.edu.sg">Centre for Development of
-            Teaching and Learning (CDTL)</a> of the National University of Singapore. TEAMMATES also
-          received funding support under the <a
-            href="https://www.google-melange.com/gsoc/homepage/google/gsoc2014"> Google Summer
-            of Code 2014</a> program.<span class="bold">TEAMMATES</span> is hosted on the <a
-            href="http://code.google.com/appengine/">Google App Engine</a>.
+          <span class="bold">TEAMMATES</span> is a joint effort by the team listed below, brought to you 
+          with the kind support of the above parties and hosted is hosted on the
```
Comment (yj-soh) [inline src/main/webapp/about.html]: "and hosted is hosted on".

Comment (junhaoyap) [inline src/main/webapp/about.html]: Oops, thanks for the catch! I'll update it soon :)

**Comment by junhaoyap** [comment]
Updated!

**Comment by damithc** [comment]
Leave the main heading as 'About us' and make 'Acknowledgements' as same level as 'Core team'? We can still keep it on top.

**Comment by junhaoyap** [comment]
Understood, I will get to it and post a new screenshot and ask for your comments again, brb!

**Comment by junhaoyap** [comment]
screen shot 2015-06-29 at 12 05 20 pm

**Comment by junhaoyap** [comment]
Ready to be looked at again, any comments on the copy?

**Comment by damithc** [comment]
The 'above parties' are no longer above. Anyway, we can get rid of that sentence entirely.

**Comment by junhaoyap** [comment]
screen shot 2015-06-29 at 12 30 50 pm

**Comment by junhaoyap** [comment]
Can take a look again :)

Does that mean that we no longer need to / want to tell visitors about being hosted on GAE anymore?

**Comment by damithc** [comment]
It's already mentioned in the landing page, so we can omit here. Actually, I don't mind having it here but it doesn't fit very well with the rest.

**Comment by junhaoyap** [comment]
Yupp, agreed. Do we want another pair of eyes to look through this before we set it to to merge?

**Comment by damithc** [comment]
Can remove `(an industry leading profiler tool for Java applications)`. Good to merge.

**Comment by junhaoyap** [comment]
Updated, will merge so that it will appear in the nex patch

**Pull request merged**